index.html

```diff
@@ -5497,6 +5506,44 @@ <h2>Base Context</h2>
 }
 </pre>
 
+      </section>
+      <section class="informative appendix">
+        <h2>Differences between Contexts and CredentialSchemas</h2>
+        <p>
+The primary purpose of a <code>credentialSchema</code> is different from the
+primary purpose of <code>@context</code>. A <code>credentialSchema</code> is
+useful for defining the data shape and type of a <code>Verifiable
+Credential</code>, whereas [[JSON-LD]] and a <code>@context</code> in a
+verifiable credential are best used only for conveying the semantics and term 
+definitions of the data.
+        </p>
+
+        <p>
+A <code>@context</code> can help you define [[JSON-LD]] semantics and term
+definitions. A <code>credentialSchema</code> is much more flexible and can help
+you define [[JSON-SCHEMA-2018]], [[SHACL]], or any other schema technology you
+want to use to further constrain the shape of the data.
+        </p>
+
+        <p>
+While it is possible to use some of [[JSON-LD]] features to allude to the
+shape of the data, it's not generally suggested to use <code>@context</code> for
+schemas related to the shape of the data model. For example, <code>"@type":
```

Comment:
suggest replace "shape of the data model" with "contents of the <code>VerifiableCredential</code>"